build.gradle

```diff
@@ -156,6 +156,19 @@ dependencies {
         exclude module: 'kryo' // use Spark's version
```

Comment:
protected uses a different version of this dependency (0.26) -- will there be a conflict?